### PR TITLE
Mar 3338.2-[Adding external ref to links]

### PR DESCRIPTION
**Pull request by Reshma-M-A**
This pull request is adding external link icon to the SFIA link. 
These are the main changes:

1. The external link css is site wide.
2. The modified webpack config to include svg files 
3. Removed the svg export in the webpack config 
4. converted some of the svg files as js

### CODE REVIEW

File: apps/marketplace/components/SellerAssessment/SellerAssessmentIntroductionStage.js [hunk 1]
```diff
@@ -22,11 +22,7 @@ const SellerAssessmentIntroductionStage = props => (
     <ul>
       <li>
         your maximum daily rate for your services in {props.meta.domain.name}. Use{' '}
-        <a
-          href="https://www.sfia-online.org/en/framework/sfia-7/busskills/level-5"
-          rel="noopener noreferrer"
-          target="_blank"
-        >
+        <a href="https://www.sfia-online.org/en/framework/sfia-7/busskills/level-5" rel="external">
```
Comment (malmas18) [inline apps/marketplace/components/SellerAssessment/SellerAssessmentIntroductionStage.js]: does this open in a new tab or window?

**Comment by tobiasbrown** [comment]
There are Zendesk links in these files.  Do you need to change them too?
* BriefSubmitted.js
* CommonMobileLinks.js
* BuyerOnboarding.js
* OpportunityInfoCard.js
* OpportunitySpecialistInfoCard.js

File: apps/marketplace/components/BuyerDashboard/BuyerDashboardHelp.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ const BuyerDashboardHelp = () => (
       <span />
       <h2 className="au-display-lg">Need a hand?</h2>
       <p>
-        <a href="/buyers-guide" rel="external">
+        <a href="/buyers-guide" target="_blank" rel="noopener noreferrer">
           Read the buyers guide
         </a>{' '}
         or <a href="/contact-us">contact us</a> - we can help you write your brief.
```
[the remark this answers is not in the document]
Comment (Reshma-M-A) [reply apps/marketplace/components/BuyerDashboard/BuyerDashboardHelp.js]: It is a case by case scenario. 
So any links that takes the user to a different domain will be opened to a new tab. This is especially the case in the brief builder

**Comment by Reshma-M-A** [comment]
> There are Zendesk links in these files. Do you need to change them too?
> 
> * BriefSubmitted.js
> * CommonMobileLinks.js
> * BuyerOnboarding.js
> * OpportunityInfoCard.js
> * OpportunitySpecialistInfoCard.js

BriefSubmitted.js - since the brief is submitted, it might not make sense to go to a new tab. 
CommonMobileLinks.js - these links might not need to go to a new tab
BuyerOnboarding.js - these links might need not need to go to a new tab
OpportunityInfoCard.js - these links might need not need to go to a new tab
OpportunitySpecialistInfoCard.js - these links might need not need to go to a new tab

**Comment by tobiasbrown** [comment]
I noticed in your changes that you've corrected a typo `noreferer`.  There may be other instances we need to update.  What about `BuyerTrainingRequirementsStage.js` and `BuyerTrainingResponseFormatsStage.js`?

**Comment by Reshma-M-A** [comment]
> I noticed in your changes that you've corrected a typo `noreferer`. There may be other instances we need to update. What about `BuyerTrainingRequirementsStage.js` and `BuyerTrainingResponseFormatsStage.js`?

Good Catch! It is fixed now.

**Pull request merged**